src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -163,12 +163,14 @@ def configure_data(self, cfg, data_cfg):  # noqa: C901
 
         logger.info("configure_data()")
         if data_cfg:
-            for subset in cfg.data:
-                if subset in data_cfg.data:
+            for subset in data_cfg.data:
+                if subset in cfg.data:
                     src_data_cfg = self.get_data_cfg(cfg, subset)
                     new_data_cfg = self.get_data_cfg(data_cfg, subset)
                     for key in new_data_cfg:
                         src_data_cfg[key] = new_data_cfg[key]
+                else:
+                    raise Exception(f"{subset} of data_cfg is not in cfg")
```

Comment:
```suggestion
                    raise ValueError(f"{subset} of data_cfg is not in cfg")
```
How about raising a specific error rather tan basic exepction?